lib/generators/administrate/install/install_generator.rb

```diff
@@ -40,6 +40,11 @@ def warn_about_invalid_models
           puts "WARNING: Unable to generate a dashboard for #{invalid_model}."
           puts "         It is not connected to a database table."
         end
+
+        unnamed_constants.each do |invalid_model|
+          puts "WARNING: Unable to generate a dashboard for #{invalid_model}."
+          puts "         It is not a valid Ruby constant."
```

Comment:
@jakecraige I... don't know.

I'm about 85% sure this is a bug in Rails, and I'm going to open a PR there to try to fix it. On the user's end, they're probably not aware they even have a non-constant class in their application, so it'll be strange for them to see a warning.

I'm considering removing this warning altogether, or changing it to something less scary-sounding, like:

```
NOTICE: Skipping dynamically generated model `#<Class:asdfasdf>`.
```

Thoughts?